src/components/header/Header.tsx

```diff
@@ -0,0 +1,27 @@
+
+import React  from 'react'
+
+import { GrFilter } from 'react-icons/gr'
+import { TbArrowsSort } from 'react-icons/tb'
```

Comment:
You can add more descriptive name to named imports (decide whether you want to implement this):
```suggestion
import { GrFilter as FilterIcon } from 'react-icons/gr'
import { TbArrowsSort as SortIcon } from 'react-icons/tb'
```